x-pack/legacy/plugins/integrations_manager/server/plugin.ts

```diff
@@ -28,11 +21,12 @@ export type PluginStart = ReturnType<Plugin['start']>;
 export class Plugin {
   constructor(initializerContext: PluginInitializerContext) {}
   public setup(core: CoreSetup) {
-    const { route } = core.http;
+    const { server } = core.http;
```

Comment:
`http.server` is there for backwards compatibility/legacy, but you can also switch to `http.registerRouter` to only rely on new platform functionality.

Reply:
just reminder that `register Router` interfaces is not stabilized yet. it will be exposed once we implement https://github.com/elastic/kibana/pull/39767

Reply:
@skaapgif lol, you’re calling out all my side quests from https://github.com/elastic/kibana/pull/39833. There was a version of it with that. I didn’t use it because it felt too early (@restrry’s point about the rfc, issues w/import, etc). Happy to try it out anytime.